apps/converter/Components/Time.js

```diff
@@ -0,0 +1,402 @@
+ 
+import React  from "zarkit/react";
+function TimeFunc(val,madeSelection_1,madeSelection_2) {
+    var result,formulvalue;
+    if (madeSelection_1 === madeSelection_2 ) {
```

Comment:
_Same here (removing a lot of `else if` statements and use an object instead)_
(copying same statement and not converting to an dynamic function is wrong)